src/components/chat/chat-panel.tsx

```diff
@@ -1,29 +1,52 @@
 import { inject, observer } from "mobx-react";
 import React from "react";
 import { BaseComponent } from "../base";
+import { getDisplayTimeDate } from "../../utilities/time";
 import { ChatPanelHeader } from "./chat-panel-header";
+import { CommentCard } from "./comment-card";
 import "./chat-panel.scss";
+import { UserModelType } from "../../models/stores/user";
 
 interface IProps {
   newCommentCount: number;
   onCloseChatPanel:(show:boolean) => void;
 }
 
+interface IState {
+  commentData: { comment: string; timePosted: string; user: UserModelType}[];
```

Comment:
Since `{ comment: string; timePosted: string; user: UserModelType}` occurs in multiple places it should be defined as an interface and referenced. Something like:
```typescript
interface ICommentData {
  comment: string;
  timePosted: string;
  user: UserModelType;
}
...
interface IState {
  comments: ICommentData[];
};
```